lib/UserInterface/Widgets/AppBar/navigate_back_appbar.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:expense_tracker/UserInterface/Theme/AppTheme.dart';
+import 'package:flutter/material.dart';
+import 'package:clay_containers/clay_containers.dart';
+
+class NavigateBackBar extends StatelessWidget implements PreferredSizeWidget {
```

Comment:
You should be naming this NeuAppBar and it can be used for all app bar requirements